MATH2269_final_project.Rmd

```diff
@@ -672,102 +697,175 @@ if(hasnt_run(trial_type)){
+
+Thinning steps will also be increased to attempt to improve the last two parameters with autocorrelation left in them (**tau** and **??**).
+
+```{r Prediction Diagnostics, echo=FALSE, fig.show = "hold", out.width = "50%", fig.align = "default", fig.cap= "A sample of the prediction model diagnostics for trial model_001_gamma_gamma_c3_a500_b500_t3.", fig.width = 10}
 
 view_prediction_diagnostics_images(trial_type)
 
 ```
 
+```{r Posterior Images, echo=FALSE, fig.show = "hold", out.width = "50%", fig.align = "default", fig.cap= "The posteriod distribution for trial model_001_gamma_gamma_c3_a500_b500_t3.", fig.width = 10}
+
+view_posterior_images(trial_type)
+
+```
+In the posterior distributions we can also see that the distributions aren't smooth indicating that there is not enough data produced by the iterations of the MCMC model.
+
+
+```{r Summary Table}
+
+view_summary_table(trial_type) %>% format_table(p_caption="", text_size = 7)
+
+```
 
 
-```{r Posterior Images, echo=FALSE, fig.show = "hold", out.width = "50%", fig.align = "default", fig.cap= "", fig.width = 10}
+#### model_002_gamma_gamma_c3_a500_b1000_t5
+
+```{r}
+
+trial_type <- "model_002_gamma_gamma_c3_a500_b1000_t5"
+
+# Parameters to use
+use_parameters <- list(number_chains = 3, 
+                       number_adaptation_steps = 500, 
+                       burn_in_steps = 1000, 
+                       thinning_steps = 5)
+
+# CHECKME: This is an expensive function, ensure it's not run when knitting, etc
+if(hasnt_run(trial_type)){
+
+    # Run JAGS model with candidate sample
+    returned_values <- setup_run_JAGS_trial(data = subsampled_data, 
+                                            predictor = "pm10", 
+                                            predictions = xPred, 
+                                            mu_list = mu,
+                                            var_list = var, 
+                                            initial_values = initial_values_list, 
+                                            params = use_parameters,
+                                            par_trial_name = trial_type, 
+                                            num_predictions = nrow(xPred))
+
+    # Reset
+    graphics.off()
+
+} else {
+  
+  print(paste0(trial_type,' already run'))
+  
+}
+
+```
+
+This model took `r view_trial_info(trial_type)$duration %>% round(1)` to run.
+
+```{r Regression Diagnostics, echo=FALSE, fig.show = "hold", out.width = "50%", fig.align = "default", fig.cap= "The regression parameter model diagnostics for trial model_002_gamma_gamma_c3_a500_b1000_t5.", fig.width=10}
+
+view_regression_diagnostics_images(trial_type)
+
+```
+
+```{r Prediction Diagnostics, echo=FALSE, fig.show = "hold", out.width = "50%", fig.align = "default", fig.cap= "A sample of the prediction model diagnostics for trial model_002_gamma_gamma_c3_a500_b1000_t5.", fig.width = 10}
+
+view_prediction_diagnostics_images(trial_type)
```

Comment:
Also repeated

Reply:
On purpose. Repeated for the second model. It is a requirement to do so!